packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -20,6 +20,7 @@ import { IProxyAdmin } from "src/universal/interfaces/IProxyAdmin.sol";
 import { IProxy } from "src/universal/interfaces/IProxy.sol";
 
 import { IAddressManager } from "src/legacy/interfaces/IAddressManager.sol";
+import { IL1ChugSplashProxy } from "src/legacy/interfaces/IL1ChugSplashProxy.sol";
```

Comment:
unused import CI check is failing

Reply:
fixed in: https://github.com/ethereum-optimism/optimism/pull/12370/commits/2f86c1a57c2275104e1d2439fbf3f4e84ac07bf0#diff-5c852d83cdaa02035a62f7aec9ff1dfd7cf9a97de6d529faeecf8d4dfeb93878R156